oozie-to-airflow/o2a.py

```diff
@@ -16,14 +16,17 @@
 import argparse
 import os
 import sys
+import logging
 
 from converter.oozie_converter import OozieConverter
 from converter.mappers import ACTION_MAP, CONTROL_MAP
+from utils.constants import CONFIGURATION_PROPERTIES
 
 INDENT = 4
 
 
 # pylint: disable=missing-docstring
+# pylint: disable=logging-too-many-args
```

Comment:
Is is required?

Reply:
You're right, after the change in logging to use t f-string it is no longer necessary. I removed it.